cap-ci/config.yaml

```diff
@@ -0,0 +1,30 @@
+jobs:
+  - deploy
+  - smoke-tests
+  - cf-acceptance-tests-brain
+  - cf-acceptance-tests
+  - sync-integration-tests
```

Comment:
Jobs list should get populated based on scenarios, instead of hardcoding it. For example, if I set `"cf-acceptance-tests": false` then the jobs list should exclude it.

Reply:
DONE.